samples/zigbee/cli/README.rst

```diff
@@ -0,0 +1,153 @@
+* Troubleshooting the existing network - reading and writing attributes, subscribing to the reports of the parameters
+* Benchmarking - measuring time needed for message to travel from one node to another
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed in `Zigbee CLI Reference`_.
+Zigbee CLI is integrated into the system shell accessible over serial connection.
+
+You can also use the external `Python CLI wrapper`_ to control Zigbee network and devices using Python language.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
+
+Use two compatible development kits programmed with this sample.
+Additionally, use either one compatible development kit programmed with :ref:`zigbee_network_coordinator_sample` for testing communication or a third compatible development kits programmed with the Zigbee CLI sample.
+
+User interface
+**************
+
+All the interactions with the application are handled using serial communication.
+See `Zigbee CLI Reference`_ for the list of available serial commands.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/zigbee/cli`
+
+|enable_zigbee_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After building the sample and programming it to your development kits, test it by performing the following steps:
+
+#. Turn on the development kits.
+#. Set up the serial connection with the development kits that are programmed with the Zigbee CLI sample.
+   You can use one of the following CLI interfaces:
+
+   * UART - For more details, see :ref:`putty`.
+   * RTT - For more details, see :ref:`testing_rtt`.
+
+#. Depending on whether you have access to a standalone Zigbee Coordinator:
+
+   * If you have a standalone Zigbee Coordinator, make sure it is able to accept new nodes.
+   * If you do not have a standalone Zigbee Coordinator, use one of the following solutions:
+
+     - Use a Zigbee Coordinator from the :ref:`zigbee_network_coordinator_sample` sample and flash it on a third board.
+     - Set up one of the CLIs to act as Zigbee Coordinator (see `Setting up CLI device as Zigbee Coordinator`_).
+
+#. By default, the CLI sample implements the Zigbee Router functionality.
+   If you have a coordinator nearby that is listening on channel 16, run the following command on boards programmed with the CLI sample (excluding the coordinator board, if you use a CLI board as coordinator):
```

Comment:
"If you have a coordinator nearby using the channel 16"? Justo listening on channel 16 seems not precise.